test/molecule-role/molecule/secrets/Readme.md

```diff
@@ -0,0 +1,81 @@
+
+Secrets configuring
+
+You need to specify path to the decrypting program by specifying `secret_backend_command`
+command.
+
+All sensitive secrets should follow pattern "ENC[keyname]" 
+
+```
+secret_backend_command: /etc/stackstate-agent/dummy_secret_feeder.sh
+
+# The StackState api key to associate your Agent's data with your organization.
+api_key: "ENC[api_key]"
+
+```
+
+stackstate-agent will pass following json request to stdin of the decrypting program,
```

Comment:
stackstate-agent will pass **the** following

Reply:
fixed